.travis.yml

```diff
@@ -1,7 +1,10 @@
 language: ruby
 rvm:
-  - 2.2.2
-  - 2.1.6
-  - 2.0.0
+  - 2.6.3
+  - 2.5.5
+  - 2.5.1
+  - 2.3.8
   - ruby-head
   - jruby-9.0.0.0
+before_install:
+  - gem install bundler --no-doc
```

Comment:
Had to add this because of a bundler error i'm getting during `bundle install`. This causes the failing build for jruby.

Reply:
Do we _need_ to support jruby?